src/requirements.txt

```diff
@@ -28,3 +28,4 @@ yapf==0.28.0
 flask_jwt_extended==3.22.0
 passlib==1.7.1
 PyYAML==5.1.2
+flask_mail
```

Comment:
I think that would be better if you add the exact version of the if the requirement that you are using. You can use pip freeze | grep flask_mail. This can help to prevent if there's a change on the library that could breake something.

Reply:
I'll include the updated requirements.txt in my next pull after doing a pip freeze, but thanks for the reminder!

Reply:
So let's close this one and wait for yours